gluten-core/src/main/scala/io/glutenproject/extension/ColumnarOverrides.scala

```diff
@@ -732,20 +732,55 @@ case class ColumnarOverrideRules(session: SparkSession)
   @transient private lazy val planChangeLogger = new PlanChangeLogger[SparkPlan]()
 
   // This is an empirical value, may need to be changed for supporting other versions of spark.
-  private val aqeStackTraceIndex = 13
+  private val aqeStackTraceIndex = 14
```

Comment:
why change to 14 ?

Reply:
this pr adds one more function `setAdaptiveContext` and move `Thread.currentThread.getStackTrace` into it.